### PR TITLE
Energy weapons are less affected by EMPs

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Energy weapons affected by EMPs will retain 2 shots. Weaker EMPs will leave more shots available.
/:cl:

Soften the counter. Make the counter less hard.
This is a var, so you can, at your discretion, make some weapons less or more affected by EMPs.

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
people who are most commonly attacked with emps that are armed with guns also have fucking batons, flashbangs, and pepperspray, which are not cucked by this.

**Comment by ChangelingRain** [comment]
sssh i'm trying to nerf every antag at once

**Comment by ChangelingRain** [comment]
sssh i'm trying to nerf every antag at once

**Comment by Supermichael777** [comment]
hahahahah this man has never tried to use pepper-spray, painfully slow flash-bangs which beep loudly, and rngbaton (disarm spam GOOOOOOOOO!)

**Comment by PKPenguin321** [comment]
:-1: What's the point of removing *some* stuns if all they need is 1 to kill you?

**Comment by Supermichael777** [comment]
tasers are hard to hit with. it can easily take 3 shots to hit a running target. How does this affect multiple type weapons?

**Comment by AdamElTablawy** [comment]
why are we NERFING emps since they're impossible to get unless antag and antags have superior murder weapons

File: code/modules/projectiles/ammunition/energy.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	caliber = "energy"
 	projectile_type = /obj/item/projectile/energy
 	var/e_cost = 100 //The amount of energy a cell needs to expend to create this shot.
+	var/emp_retained_shots = 2
```
Comment (Fox-McCloud) [inline code/modules/projectiles/ammunition/energy.dm]: magic numberssss

**Comment by Supermichael777** [comment]
iorn+uranium is not hard to get

plus emp nades are in some maints

**Comment by jughu** [comment]
Next pr be Tasers will teslabolt now and instantly kill you

**Comment by PKPenguin321** [comment]
>iorn+uranium is not hard to get

Lie

File: code/modules/projectiles/ammunition/energy.dm (hunk 1 above)
Comment (JJRcop) [inline code/modules/projectiles/ammunition/energy.dm]: it's not a magic number if it's an editable var

**Comment by Anonmare** [comment]
You can buy EMP grenades from cargo after multitooling and then ordering the spec OPs crate as I recall

File: code/modules/projectiles/ammunition/energy.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/projectiles/ammunition/energy.dm]: It's not magic if it's a var!

**Comment by ExcessiveUseOfCobblestone** [comment]
when are we removing EMPs so people stop acting like they're a valid balance component?

**Comment by XDTM** [comment]
Or adding a few so they actually are

**Comment by jughu** [comment]
@anonmare thats emagging

**Comment by KorPhaeron** [comment]
I hate emps but I dont understand the PR because of what PKP said

**Comment by Cebasiz** [comment]
>oh look an EMP nerf PR that's cool borgs deserve a buff
>all it does is change tasers

rip

**Comment by jughu** [comment]
>instastuns buff

**Comment by kilkun** [comment]
Hahahahahahahahahaha...

oh wait, this isn't a meme PR, let me laugh even harder

_**AHAHAHAHAHA**_

**Comment by kilkun** [comment]
There's absolutely nothing wrong with EMPs. Learn to improvise you shitters.

**Comment by optimumtact** [comment]
No consensus and cheridan didn't like this idea

**Comment by kevinz000** [comment]
DAMNIT CITRUS

**Pull request closed without merging**